Jenkinsfile

```diff
@@ -907,6 +907,70 @@ pipeline {
       }
     }
 
+    stage('L2: NMT with HuggingFace') {
+      when {
+        anyOf{
+          branch 'main'
+          changeRequest target: 'main'
+        }
+      }
+      failFast true
+      parallel {
+        stage('L2: NMT Pretrained HF Encoder') {
+            steps {
+              sh 'cd examples/nlp/machine_translation && \
+              python enc_dec_nmt.py \
+              --config-path=conf \
+              --config-name=huggingface \
+              model.shared_tokenizer=False \
+              model.encoder_tokenizer.library=huggingface \
+              model.encoder.library=huggingface \
+              model.encoder.model_name=bert-base-uncased \
```

Comment:
minor comment: can we use bert-base-cased in examples and CI. This is because training data is cased and it is a good idea to have cased model for NMT.

Reply:
Yes, good idea. I'll make the change.